modules/clightning-plugins/trustedcoin.nix

```diff
@@ -0,0 +1,26 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.clightning.plugins.trustedcoin; in
+{
+  options.services.clightning.plugins.trustedcoin = {
+    enable = mkEnableOption "Trustedcoin (clightning plugin)";
+    package = mkOption {
+      type = types.package;
+      default = config.nix-bitcoin.pkgs.trustedcoin;
+      defaultText = "config.nix-bitcoin.pkgs.trustedcoin";
+      description = mdDoc ''
+        The package providing trustedcoin binaries. Trustedcoin will try to
+        use a bitcoind as a main trusted source for getting on-chain block
+        data. If this fails, it will use a trustedcoin providers instead.
+      '';
```

Comment:
Nit: I would either remove everything but the first sentence (since the description of the package options seems a bit like the wrong location for this explanation) or also add the line "This isn't what you should be doing, but sometimes you may need it." from the trustedcoin README.